src/fuzzing/README.md

```diff
@@ -1,6 +0,0 @@
-Fuzzing
-=======
-
-    CC=afl-gcc meson --default-library=static ../
-    AFL_HARDEN=1 ninja
-    afl-fuzz -m 300 -i ../src/fuzzing/smbios -o src/smbios/findings ./src/fwupdmgr smbios-dump @@
```

Comment:
I think you'll still want to have a README here sitting around explaining the meson targets, no?

Reply:
I did ponder moving it up to the top level README.md -- all the targets are now `fuzz-` prefixed so it's all self explanatory.

Reply:
But how do you discover their existence?  I guess that's what needs to be documented somewhere.  It seems like maybe `meson introspect $DIR --targets`, but that seems too verbose.

Reply:
/me does `ninja fuzz<tab>`...

Reply:
Ah, that seems to work both with bash and zsh for me too using `ninja -C build` even.  But yeah not at all discoverable otherwise, so I would say it should be mentioned somewhere :)

Reply:
How about the follow up commit that moves it all the to the toplevel?